packages/plugin/tests/require-id-when-available.spec.ts

```diff
@@ -53,6 +65,11 @@ ruleTester.runGraphQLTests('require-id-when-available', rule, {
     },
   ],
   invalid: [
+    {
```

Comment:
I'm not sure if this one should be invalid. Was there some defect with the previous solution that I'm not following. Thanks!

Reply:
Should be moved to `valid`, you are right. 
The thing here: we need to test both edges of usage: consumed clients, and parent selection sets (in case of inline fragments). I will fix that soon, thank you!

Reply:
Fixed and available in `@graphql-eslint/eslint-plugin@0.5.0`. Thanks @henryqdineen !